articles/event-hubs/event-hubs-faq.yml

```
@@ -150,6 +150,11 @@ sections:
           Is there a cost associated when I turn on the auto-inflate feature?
         answer: |
           There's **no cost** associated with this feature. 
+          
+      - question: |
+          Can Zone Redundancy be enabled for existing Event Hub Namespace?
+         answer: |
```

```suggestion
        answer: |
```
Removing space to try to clear error.